build.gradle.kts

```diff
@@ -6,6 +6,7 @@ plugins {
   id("org.hypertrace.ci-utils-plugin") version "0.3.0"
   id("org.hypertrace.publish-plugin") version "1.0.4"
   id("org.owasp.dependencycheck") version "8.4.0"
+  id("com.diffplug.spotless") version "7.0.0"
```

Comment:
Rather than using spotless directly you could just use the last version of this plugin to have been published to dogfood it a bit.